### DISCUSSION TITLE
Ability for Admin / TA to create multiple group for group assignment

### DISCUSSION THREAD

**Discussion by jnmwangi** [community/community#33581] [GitHub Education]
This would be a great addon where, as an Admin/TA I can create multiple groups for a group assignment for students to join once they access the assignment link.

**Comment by RyzenFromFire** [comment]
Seconding this. Currently, there seems to be no way for an admin/TA to create groups. A student must create a team when they accept the assignment. Allowing a TA to create teams based on the roster would be a welcome addition (drag and drop into different boxes, perhaps). 

There is a way to circumvent this, if you as the admin/TA go to the assignment link and accept it yourself (you may want to add yourself to the roster). You can then create a team, then remove yourself from it from the organization panel. Repeat if necessary.
